wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/unterbrechungsuhrzeit/UnterbrechungsUhrzeitDTO.java

```diff
@@ -0,0 +1,13 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.rest.unterbrechungsuhrzeit;
+
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+public record UnterbrechungsUhrzeitDTO(@NotNull String wahlbezirkID,
+                                       LocalDateTime unterbrechungsUhrzeit) {
```

Comment:
Zur Korrektheit im in OpenAPI-Spec

```suggestion
                                       @NotNull LocalDateTime unterbrechungsUhrzeit) {
```